spec/models/v2/security_guide_spec.rb

```diff
@@ -3,4 +3,23 @@
 require 'rails_helper'
 
 describe V2::SecurityGuide do
+  subject { described_class.new(ref_id: 'xccdf_org.ssgproject.content_benchmark_RHEL-7') }
```

Comment:
https://www.betterspecs.org/#data
https://www.betterspecs.org/#factories

```suggestion
  subject { FactoryBot.create(:v2_security_guide, os_major_version: os_version) }
```